pyproject.toml

```diff
@@ -19,28 +19,25 @@ exclude = '''
 '''
 
 [build-system]
-requires = [
-  "setuptools >= 68.0",
-  "wheel >= 0.38.4",
-  "scikit-build >= 0.17.6",
-  "cmake >= 3.28.1",
-]
-build-backend = "setuptools.build_meta"
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
 
-[tool.setuptools.dynamic]
-version = { attr = "arrayfire_wrapper.__version__" }
+#[tool.setuptools.dynamic]
+#version = { attr = "arrayfire_wrapper.__version__" }
```

Comment:
It doesn't work because you switched the `build-backend`. Let's look through the scikit_build_core to find an alternative or, maybe, it is possible to use multiple backend building tools